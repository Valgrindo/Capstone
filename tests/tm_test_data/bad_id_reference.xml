<!-- Test that a component defined separately from a command with a non-matching ID fails -->
<commands>

    <command name="TO_RESOLVE">
        <component indicator="SPEECHACT" type="SA_TELL">
            <role name="CONTENT">
                <!-- This component sis defined externally -->
                <component from_id="MISSING-ID"/>
            </role>
        </component>
    </command>

    <!-- And here is the definition -->
    <component id="BAD-ID" indicator="BARE" type="FRUIT" word="APPLE"/>

</commands>