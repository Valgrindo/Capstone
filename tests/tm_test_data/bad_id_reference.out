Cannot resolve external component ID MISSING-ID